packages/core/echo/echo-schema/src/automerge/automerge-db.ts

```diff
@@ -388,6 +391,27 @@ export class AutomergeDb {
       this._automergeDocLoader.onObjectBoundToDocument(docHandle, objectId);
     }
   }
+
+  private _objectsForNextUpdate = new Set<string>();
+  private readonly _updateScheduler = new UpdateScheduler(
+    this._ctx,
+    async () => {
+      const ids = [...this._objectsForNextUpdate];
+      this._objectsForNextUpdate.clear();
+      this._emitUpdateEvent(ids);
+    },
+    {
+      maxFrequency: 10,
```

Comment:
isn't it too aggressive to potentially delay an update by 100 ms? I'd not go higher than 16ms (60hz)?

Reply:
👍